.github/workflows/continuous.yml

```diff
@@ -145,6 +145,7 @@ jobs:
       - name: Compile PaddlePaddle JNI
         run: ./gradlew :paddlepaddle:paddlepaddle-native:compileJNI
       - name: Integration tests
+        # MXNet integration test on Github Action fail due to memory limitation on Windows
         run: |
-          ./gradlew :paddlepaddle:paddlepaddle-engine:test :paddlepaddle:paddlepaddle-model-zoo:test
-          ./gradlew :onnxruntime:onnxruntime-engine:test
+          ./gradlew --rerun-tasks -Dai.djl.default_engine=PyTorch build
```

Comment:
Do rerun on all tasks will make the test time tripled. I would still suggest we do test in parallel and for the sake of clearance.

Reply:
1. The build itself is very fast, less than 1 minute, especially for TF and PT integration test
2. The majority time spend is on spin up the machine, download engine native binary and dependencies. 
3. Windows build is run parallel with Linux and mac, it should not increase total continuous build time